tests/unit/main_loop/test_main_loop.py

```diff
@@ -72,9 +72,9 @@ def test_wrapper_calls_function():
     flag = False
 
     async def test_coro(arg1, arg2):
+        nonlocal flag
         assert arg1 == 'arg1'
         assert arg2 == 'arg2'
-        nonlocal flag
         flag = True
```

Comment:
Another example where `nonlocal` is required. Because the variable is a primitive, not an object, we have to rebind it rather than modify it in place.